test/features/caseView_printButton.feature

```diff
@@ -0,0 +1,23 @@
+@functional @caseView
+Feature: Set of scenarios testing case view functionality
+
+  Background:
+    Given I have logged in
+
+  Scenario: print button visible on case view when configured
+    Given the following definition for 'case tape'
+      | ID              | Name                 | PrintableDocumentsUrl                                                                     |
+      | AllDataTypes2   | All Field Data Types | http://localhost:4452/callback/jurisdictions/AUTOTEST1/case-types/AllDataTypes2/documents |
```

Comment:
anything better we can think to avoid having the print url in the scenario?
For example we could just check whether there is any value (i.e. PrintableDocumentsUrl is not empty)

Reply:
👍